q2cli/builtin/tools.py

```diff
@@ -195,7 +195,89 @@ def peek(path):
         click.echo(metadata.format)
 
 
-@tools.command('inspect-metadata',
+@tools.command(name='cast-metadata',
+               short_help='Designate metadata columns as numerical'
+                          ' or categorical.',
```

Comment:
Thanks @thermokarst! What are your thoughts on including a link to the Metadata docs page with the supported column types? Something like 'supported column types can be found here: https://docs.qiime2.org/2021.4/tutorials/metadata/?highlight=qiime%20tools#column-types' so that users have an easy reference if they are unsure of the column types they can use.

Reply:
I think including a link in the `short_help` text will make the prose a bit unreadable, unfortunately.

Reply:
To be clear, I do like that idea in the full help text, although you'll need to sort out how to handle the URL versioning - we don't want to make a maintenance issue here when releasing new version of QIIME 2.

Reply:
@thermokarst a better solution that we pushed up is just including docs.qiime2.org as a general push in the right direction for folks who are unsure. Let me know what you think of that formatting!